src/utils/filplus-edition.ts

```diff
@@ -1,25 +1,44 @@
-export type FilPlusEdition = {
+import { DateTime } from 'luxon';
+
+export type FilplusEditionConfig = {
   id: number;
   start?: number;
   end?: number;
   lowReplicaThreshold: number;
   highReplicaThreshold: number;
 };
 
+export type FilPlusEdition = {
+  startDate: Date;
+  endDate: Date;
+  isCurrent: boolean;
+} & FilplusEditionConfig;
+
 // prettier-ignore
-export const filPlusEditions: FilPlusEdition[] = [
+export const filPlusEditions: FilplusEditionConfig[] = [
   { id: 4, end: 1709251200, lowReplicaThreshold: 4, highReplicaThreshold: 10 }, // < 1 March 2024
   { id: 5, start: 1709251200, end: 1752192000, lowReplicaThreshold: 4, highReplicaThreshold: 10 }, // 1 March 2024 - 11 July 2025
   { id: 6, start: 1752192000, lowReplicaThreshold: 4, highReplicaThreshold: 8 }, // >= 11 July 2025
 ];
 
+const getFilPlusEditions = (): FilPlusEdition[] => {
```

Comment:
I dont think this function is necesary. Checking if the given edition is the current one is simple: `edition === getCurrentFilPlusEdition()` and `startDate`, `endDate` can already be encoded in `export const filPlusEditions: FilplusEditionConfig`